src/EnergyPlus/Psychrometrics.hh

```diff
@@ -1292,14 +1292,25 @@ namespace Psychrometrics {
     }
 
     inline Real64 PsyDeltaHSenFnTdb2W2Tdb1W1(Real64 const TDB2, // dry-bulb temperature at state 2 {C}
-                                             Real64 const dW2,  // humidity ratio at  at state 2
-                                             Real64 const TDB1, // dry-bulb temperature at  at state 1 {C}
-                                             Real64 const dW1   // humidity ratio  at state 1
+                                             Real64 const dW2,  // humidity ratio at state 2
```

Comment:
Ok. renamed the variables.

Reply:
@Nigusse Did you push up your latest changes? These are still named `dW`. Also, these special Psych functions need some comments to explain how they are derived. It appears that these are based on `PsyHFnTdbW` and then algebraically simplified to the delta H case to reduce computation time. Some comments referencing the base function name would be helpful, otherwise, it's not clear where these equations and constants come from. A comment to the effect of this is equivalent to `PsyHFnTdbW(TDB2, min(W1, W2)) - PsyHFnTdbW(TDB1, min(W1,W2))` would be adequate. Here and in the other similar functions. Also with a brief comment explaining the use case for each one (delta across a coil, supply into a zone, etc.).

Reply:
I definitely renamed these variables elsewhere but may be I missed some of them.  I will renam the remaining variables and add comments how the two functions were derived and where the constant coefficients came from.

Reply:
Done! I will test the unit test suite locally and push the branch shortly.